activerecord/lib/active_record/railties/databases.rake

```diff
@@ -176,7 +176,7 @@ db_namespace = namespace :db do
   task setup: ["db:schema:load_if_ruby", "db:structure:load_if_sql", :seed]
 
   desc "Loads the seed data from db/seeds.rb"
-  task :seed do
+  task seed: [:environment, :load_config] do
```

Comment:
Comparing this to what's currently on master a0061d2389a178b093f0d3f64f58236ffbe088e0, I think all that's needed is `task seed: :load_config do` because `:load_config` depends on `:environment`.

Reply:
true point, will update & resolve conflict.. thanks for the feedback